templates/dashboard/index.html.twig

```diff
@@ -27,7 +27,7 @@
                                           <tr {% for user in report.seen %}{% else %}class="table-warning"{% endfor %}>
                                             <th scope="row">{{ report.id }}</th>
```

Comment:
Id is just an internal number and holds little value. I am even thinking of removing it from public view all together. please hide on smaller screens

Reply:
OK.